app/generic/connections/connections.py

```diff
@@ -172,18 +175,18 @@ async def accept_invitation(
         the invitation object obtained from create_invitation.
     """
 
-    conn_record = await aries_controller.connection.receive_invitation(
+    connection_record = await aries_controller.connection.receive_invitation(
         body=body.invitation,
         auto_accept=True,
         alias=body.alias,
     )
-    return conn_record_to_connection(conn_record)
+    return conn_record_to_connection(connection_record)
 
 
 @router.get("/", response_model=List[Connection])
 async def get_connections(
     aries_controller: AcaPyClient = Depends(agent_selector),
-):
+) -> List[Optional[Connection]]:
```

Comment:
I'm not sure this is the correct type. List[Optional[Connection]] means there could be None values in the list ([None, Conn, None, None]), but that's not the case right (and otherwise _shouldn't_ be the case).

There is the possiblity of returning an empty list, but in that case the type `List[Connection]` still is correct